src/store/conversationsStore.js

```diff
@@ -95,6 +95,27 @@ const actions = {
 		if (!currentUser.uid) {
 			currentUser = getCurrentUser()
 		}
+
+		if (!currentUser || !currentUser.uid) {
+			// Guests can not fetch the full participant list, but at least the
+			// participants store can be populated with the user ID, session ID,
+			// participant type and call status of the users from the
+			// conversation data.
+			context.dispatch('purgeParticipantsStore', conversation.token)
+
+			for (const participantId in conversation.participants) {
```

Comment:
This is going to be removed in one of the next versions. Is you code depending on it?
maybe we should instead  allow guests to access the proper endpoint instead.

Reply:
> This is going to be removed in one of the next versions. Is your code depending on it?

It was needed when the filtering was done in the clients, but now that it is done in the server it should not be needed. I will check and remove it if is not needed.

> maybe we should instead allow guests to access the proper endpoint instead.

I think it would make sense. With the external signaling server guests can get the full list of active participants from the `participants` event message, and they can get the list of users (without display name) from the conversation data. In the end they already have access to almost all the data that the participants endpoint provides, so I think that preventing their access to that endpoint does not provide too much value. Anyway, this would be something for a different pull request ;-)